### PR TITLE
Update the file types accepted to match those supported by PM webapp

### LINKED ISSUE #23: Accept the new files types supported by PM
Ref : https://gitlab.com/exotic-matter/ftl-app/-/merge_requests/155

File types list:
```
    '.pdf',
    '.txt',
    '.rtf',
    '.doc',
    '.xls',
    '.ppt',
    '.docx',
    '.xlsx',
    '.pptx',
    '.odt',
    '.odp',
    '.ods',
```

### PR DESCRIPTION
**Pull request by fdaveine**
Fix #23

### CODE REVIEW

File: src/renderer/components/HomePage/ImportTab.vue [hunk 1]
```diff
@@ -160,8 +161,17 @@
 
         // Store files to import in store if needed (not needed when documents are recover from a previous session)
         if (vi.files.length > 0 || vi.filesInsideFolder.length > 0){
-          // we filter filesInsideFolder to get only pdf files
-          const filteredFilesInsideFolder = vi.filesInsideFolder.filter(file => file.type === 'application/pdf');
+          // we filter filesInsideFolder to get only supported files
+          const filteredFilesInsideFolder = vi.filesInsideFolder.filter(file => {
+            // check if file name contains a dot (no dot = no extension)
+            if (file.name.includes(".")){
+              let fileExtension = file.name.split('.');
```
Comment (antoinelibert) [inline src/renderer/components/HomePage/ImportTab.vue:168]: Maybe use https://nodejs.org/api/path.html#path_path_extname_path ?

File: src/renderer/components/HomePage/ImportTab.vue [hunk 2]
```diff
@@ -160,8 +161,17 @@
...
+              fileExtension = fileExtension[fileExtension.length-1];
+              return this.supportedFileExtensions.includes(`.${fileExtension}`);
```
Comment (antoinelibert) [inline src/renderer/components/HomePage/ImportTab.vue:170]: Handling of uppercase/mixed extension?

File: src/renderer/i18n/messages.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ export default {
       startImportButton: 'Start import',
       startImportLoadingButton: 'Checking metadata...',
       filePathLabel: 'File Path*',
-      filePathTooltip: 'Full path of PDF files to import (required)',
+      filePathTooltip: 'Full files path to import (required)',
```
Comment (antoinelibert) [inline src/renderer/i18n/messages.js:103]:
```suggestion
      filePathTooltip: 'Full filepaths to import (required)',
```

File: src/renderer/components/HomePage/ImportTab.vue (hunk 1 above)
Comment (fdaveine) [reply src/renderer/components/HomePage/ImportTab.vue]: Fixed by 2bae59c

File: src/renderer/components/HomePage/ImportTab.vue (hunk 2 above)
Comment (fdaveine) [reply src/renderer/components/HomePage/ImportTab.vue]: Fixed by 2bae59c

**Pull request merged**